examples/ios/swift/ListSwiftUI/Models/Recipe.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import RealmSwift
+import SwiftUI
+
+final class Recipe: Object {
+    @objc dynamic var id: String = UUID.init().uuidString
+    @objc dynamic var name: String = ""
+    let ingredients = RealmSwift.List<Ingredient>()
+
+    override class func primaryKey() -> String? {
+        "id"
+    }
+
+    class func new(name: String, ingredients: Ingredient...) -> Recipe {
+        let recipe = Recipe()
+        recipe.name = name
+        recipe.ingredients.append(objectsIn: ingredients)
+        return recipe
+    }
+}
+
+final class Recipes: Object {
+    @objc dynamic var id: String = ""
+    let recipes = RealmSwift.List<Recipe>()
+
+    static let shared: Recipes = ({
+        let realm = try! Realm.init(configuration: .init(deleteRealmIfMigrationNeeded: true))
+
+        guard let uuid = UIDevice.current.identifierForVendor,
```

Comment:
This should just be an `if` rather than a `guard`.

The use of the device identifier as the primary key complicates things and doesn't do anything useful. For a non-synchronized Realm it can just always be 0. A synchronized Realm would instead want to use something based on the user id.